plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -331,10 +331,12 @@ protected ResultSet getColumns(JdbcTableHandle tableHandle, DatabaseMetaData met
             throws SQLException
     {
         RemoteTableName remoteTableName = tableHandle.getRequiredNamedRelation().getRemoteTableName();
+        final String escapePattern = metadata.getSearchStringEscape();
+        final String tableName = remoteTableName.getTableName();
         return metadata.getColumns(
                 remoteTableName.getCatalogName().orElse(null),
```

Comment:
fixed (force push removed previous warning by @findinpath about unnecessary escaping of the catalog name)